### PR TITLE
v1.4.1

### PR DESCRIPTION
**Pull request by MatthewFlamm**

### CODE REVIEW

**Comment by lymanepp** [comment]
Hi @MatthewFlamm,

I'm writing a custom integration that uses pynws directly to get the humidity from the detailed weather forecast. Since the HA core team doesn't seem to be interested in adding humidity to the `Forecast` class, I resorted to creating my own internal provider model to be compatible with multiple weather services (more than just USA). HA appears to block attempts by my component to install a newer version of pynws (that's probably not a bad thing).

TL; DR, would it be possible to bump the HA supported version of pynws to v1.4.1 in the near future?
